test/config/install_default.json

```diff
@@ -30,6 +30,27 @@
       "proxy": {
         "host": "",
         "port": ""
-      }
+      },
+      "os-install": [
+        {"workflow": "Graph.InstallESXi", "version": "6.0", "path": "/repo/esxi/6.0", "kvm": false},
+        {"workflow": "Graph.InstallCentOS", "version": "6.5", "path": "/repo/centos/6.5", "kvm": false},
+        {"workflow": "Graph.InstallUbuntu", "version": "trusty", "path": "/repo/ubuntu", "kvm": false},
+        {"workflow": "Graph.InstallESXi", "version": "5.5", "path": "/repo/esxi/5.5", "kvm": false},
```

Comment:
The order was selected to run the three PR gate OS installs first, for example in the case of only three nodes available.

Comments are not supported in JSON files. I can add that to the comment section in the test file.